### PR TITLE
[Event Date Api] Merge country data to Earth

### PR DESCRIPTION
**Pull request by chejennifer**
- the event date cache only goes up to Country level so need to merge dates when requesting Earth or continents.

### CODE REVIEW

**Review by shifucun — APPROVED** [review]

File: internal/server/recon/golden/bulk_find_entities/result.json [hunk 1]
```diff
@@ -26,10 +26,7 @@
       ]
     },
     {
-      "description": "mmountan view",
-      "dcids": [
-        "geoId/0649670"
-      ]
+      "description": "mmountan view"
```
Comment (shifucun) [inline internal/server/recon/golden/bulk_find_entities/result.json:29]: this test is flaky due to maps api, maybe we should remove it @Spaceenter ?

Comment (Spaceenter) [reply internal/server/recon/golden/bulk_find_entities/result.json:29]: Right, Maps Place API is flaky... Do we completely remove the golden test for (bulk)findEntities?
Also add @pradh for visibility.

**Review by Spaceenter — APPROVED** [review]

File: internal/server/recon/golden/bulk_find_entities/result.json (hunk 1 above)
Comment (shifucun) [reply internal/server/recon/golden/bulk_find_entities/result.json:29]: Sure, let's remove it for now to unblock.

Comment (chejennifer) [reply internal/server/recon/golden/bulk_find_entities/result.json]: done - removed

**Pull request merged**